### PR TITLE
[bitnami/parse] Add tests and publish using VIB

### PR DESCRIPTION
**Pull request by joancafom**
### Description of the change

The main objective of this PR is to publish our Bitnami Parse Helm Chart using VMware Image Builder. In order to do that, several changes are included:

- Adding the asset to the publishing workflow
- Increasing the existing test coverage of the asset by adding Cypress tests.
- Increasing the existing test coverage of the asset by adding GOSS tests.

### Benefits

- Ensuring higher quality of the Helm charts.
- Increased pool of assets completely handled by VMware Image Builder.

### Possible drawbacks

Automated tests could show to be potentially flaky. 

### Applicable issues

NA

### Additional information

Tested in https://github.com/joancafom/charts/pull/77

### Checklist

- [x] Title of the pull request follows this pattern [bitnami/<name_of_the_chart>] Descriptive title
- [x] All commits signed off and in agreement of [Developer Certificate of Origin (DCO)](https://github.com/bitnami/charts/blob/master/CONTRIBUTING.md#sign-your-work)

### CODE REVIEW

**Review by agomezmoron — APPROVED** [review]
LGTM

File: .vib/parse/goss/dashboard/goss.yaml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    exists: true
+    filetype: directory
+    mode: "2777"
+    owner: root
+  /opt/bitnami/parse-dashboard/config.json:
+    exists: true
+    filetype: file
+    mode: "0644"
+    contains:
+      - /masterKey.*{{ .Vars.server.masterKey }}/
```
Comment (FraPazGal) [inline .vib/parse/goss/dashboard/goss.yaml:12]: A bit trivial, but we could drop this check. We are already checking it on the API request.

**Review by FraPazGal — COMMENTED** [review]
Thanks for the PR @joancafom! I added a couple of minor comments, please check them when possible.

File: .vib/parse/goss/dashboard/goss.yaml (hunk 1 above)
Comment (joancafom) [reply .vib/parse/goss/dashboard/goss.yaml:12]: Actually no, the API request checks the **Server** component is configured with the _masterKey_. This is checking the **Dashboard**, for which we can't implement any Cypress tests at the moment.

This way we ensure both have matching values.

File: .vib/parse/goss/server/goss.yaml [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    filetype: directory
+    mode: "2775"
+    owner: root
+  /opt/bitnami/parse/config.json:
+    exists: true
+    filetype: file
+    mode: "0644"
+    contains:
+      - /port.*{{ .Vars.server.containerPorts.http }}/
+      - /database.*{{ .Vars.mongodb.auth.username }}.*{{ .Vars.mongodb.auth.password }}.*{{ .Vars.mongodb.auth.database }}/
```
[the remark this answers is not in the document]
Comment (joancafom) [reply .vib/parse/goss/server/goss.yaml:13]: The DB credentials might be okay (as you pointed out, that is why the app starts), but that does not mean that the app is correctly picking up the custom values that we provided.

File: .vib/parse/goss/dashboard/goss.yaml (hunk 1 above)
Comment (FraPazGal) [reply .vib/parse/goss/dashboard/goss.yaml:12]: My bad then, thanks for the clarification!

**Review by FraPazGal — APPROVED** [review]

**Pull request merged**